packages/core/docs/guide/product-catalog.md

```diff
@@ -1,112 +1,206 @@
 
-You can fetch all the data needed for a Product Details Page such as products, sorting options, filters and pagination with a single call of a`search` method. The search result will be saved in `result` object.
+`useFacet` composable is used primarily on the Category Page to display products matching specified filters.
 
-You can extract certain information from `result` object using `facetGetters`:
+Use the `search` method to fetch the products. The response is available in the `result` object.
 
-```ts
-setup (props, context) {
-  const { result, search, loading } = useFacet();
+```vue
+<script>
+import { useFacet } from '{INTEGRATION}'
+import { onSSR } from '@vue-storefront/core'
 
-  // the products that were found
-  const products = computed(() => facetGetters.getProducts(result.value));
+export default {
+  setup () {
+    const { result, search } = useFacet();
 
-  // corresponding category tree
-  const categoryTree = computed(() => facetGetters.getCategoryTree(result.value));
+    onSSR(async () => {
+      // This example shows only some of the available parameters.
+      await search({
+        categorySlug: '',
+        rootCatSlug: '',
+        term: '',
+        page: 1,
+        itemsPerPage: 10,
+        sort: ''
+      });
+    })
+    
+    return {
+      result
+    };
+  }
+}
+</script>
+```
 
-  // related breadcrumbs
-  const breadcrumbs = computed(() => facetGetters.getBreadcrumbs(result.value));
+For a full list of parameters, please refer to the [AgnosticFacetSearchParams](../core/api-reference/core.agnosticfacetsearchparams) interface.
 
-  // sort options
-  const sortBy = computed(() => facetGetters.getSortOptions(result.value));
+## Accessing catalog data
 
-  // available facets (grouped)
-  const facets = computed(() => facetGetters.getGrouped(result.value));
+Once data is loaded using `useFacet`, access it using `facetGetters`. Get the data such as products, sorting and filtering options, pagination, and much more.
 
-  // pagination options
-  const pagination = computed(() => facetGetters.getPagination(result.value));
+```vue
+<script>
+import { facetGetters } from '{INTEGRATION}'
+import { onSSR } from '@vue-storefront/core'
 
-  onSSR(async () => {
-    // triggering a search based on criteria available in url query eg. ?colo=red&sortBy=latest
-    await search(searchParams);
-  });
+export default {
+  setup () {
+    // Load products as shown in the first example above
+
+    const breadcrumbs = computed(() => facetGetters.getBreadcrumbs(result.value));
+    const sortBy = computed(() => facetGetters.getSortOptions(result.value));
+    const categoryTree = computed(() => facetGetters.getCategoryTree(result.value));
+    const facets = computed(() => facetGetters.getGrouped(result.value));
+    const products = computed(() => facetGetters.getProducts(result.value));
```

Comment:
If we load `products` as in the first example above, then `products` is already declared.